Chat App/BuildConfig/pubKeys.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
```

Comment:
Maybe instead of creating a different property list for the keys, why not creat a xcconfig for each environment and put the keys there

Reply:
Added xconfig files for debug and release environments. Committed in the next push.